instat/dlgTransformClimatic.vb

```diff
@@ -120,6 +119,7 @@ Public Class dlgTransformClimatic
         ucrInputEvaporation.SetLinkedDisplayControl(lblWBEvaporation)
 
         ucrInputColName.SetParameter(New RParameter("result_name"))
+        ucrInputColName.SetName("moving_" & ucrInputSum.cboInput.SelectedItem)

```

Comment:
When you opened the dialog, the name would automatically be "moving", not "moving_sum". This, along with deleting line 152 (in SetDefaults) fixes this bug